crates/bevy_pbr/src/material.rs

```diff
@@ -507,6 +508,7 @@ pub struct MaterialProperties {
     pub alpha_mode: AlphaMode,
     /// Add a bias to the view depth of the mesh which can be used to force a specific render order
     /// for meshes with equal depth, to avoid z-fighting.
```

Comment:
```suggestion
    /// for meshes with similar depth, to avoid z-fighting.
```

nit: change this line to match the change in documentation for `fn depth_bias(&self)`